rfc6962-bis.xml

```diff
@@ -1103,7 +1103,7 @@ messages.
         <t>
           Note that this message is not signed -- the <spanx style="verb">entries</spanx> data can be
 verified by constructing the Merkle Tree Hash corresponding to a retrieved
-STH. All leaves MUST be v1 or v2. However, a compliant v1 client MUST NOT construe an
+STH. All leaves MUST be v1 or v2. However, a compliant v2 client MUST NOT construe an
```

Comment:
"All leaves MUST be v1 or v2".  Does that make sense?  Or do we need to require that (in order to conform to this version of the specification) all leaves MUST be v2?  (i.e. each log is either entirely v1 or entirely v2).